specification/cognitiveservices/Language.AnalyzeText/models/summarization.tsp

```diff
@@ -107,7 +107,7 @@ model AbstractiveSummarizationLROTask extends AnalyzeTextLROTask {
   kind: AnalyzeTextLROTaskKind.AbstractiveSummarization;
 
   /** Parameters for the Abstractive Summarization task. */
-  parameters: AbstractiveSummarizationTaskParameters;
+  parameters?: AbstractiveSummarizationTaskParameters;
```

Comment:
Did this change in a newer version of the service? Or was it incorrectly defined?

Reply:
Incorrectly defined.